mne_realtime/tests/test_fieldtrip_client.py

```diff
@@ -124,6 +124,7 @@ def test_fieldtrip_rtepochs(free_tcp_port, tmpdir):
         kill_signal.put(False)  # stop the buffer
 
 
+@requires_neuromag2ft
```

Comment:
this should be ported over from mne and the decorator can be deprecated there since it was only used for the realtime

Reply:
I agree, can you make a PR?

Reply:
it's all in your very capable hands @jasmainak :)

Reply:
I want to have this PR merged with the bare minimum that is needed so I can work on it during upcoming coding sprint

Reply:
Feel free to open an issue though

Reply:
it would be nice if you could include this fix as it would make for a complete PR. it would also help clean up the code on mne-python but its next release.

Reply:
We just released :) Also I don't think we'd really need a deprecation cycle to get rid of this as it's an undocumented testing fixture that I doubt has very widespread use (probably only here and in MNE). I'm fine with killing it at the sprint next week separately if need be.